backend/database/models/user.py

```diff
@@ -78,5 +78,9 @@ class User(db.Model, UserMixin, CrudMixin):
 
     role = db.Column(db.Enum(UserRole))
 
+    phone_number = db.Column(
+        CI_String(14, collate="NOCASE"), nullable=False, server_default=""
```

Comment:
Since phone number is optional, we should allow this column to be empty. So I think we can just use `phone_number = db.Column(db.Text)`

Reply:
Done